src/items/extern-crates.md

```diff
@@ -16,6 +17,9 @@ the `crateid` attributes that were declared on the external crate when it was
 compiled. If no `crateid` is provided, a default `name` attribute is assumed,
 equal to the [identifier] given in the `extern crate` declaration.
 
+The `self` crate may be imported which creates a binding to the current crate.
```

Comment:
In the case of macros, I'm not sure whether the "current crate" always means the crate for the call-site or the def-site of the macro... I guess until you can use `Span::def_site` on stable it doesn't matter?

Reply:
I believe it is always call-site. I think the new macro-rule documentation explains this in the hygiene section pretty well ("all identifiers referred to in a macro are expanded as-is, and are looked up at the macro's invocation site."). Do you think it needs extra clarification here?

Reply:
> Do you think it needs extra clarification here?

No. =P